crates/uv-dev/src/generate_json_schema.rs

```diff
@@ -3,12 +3,24 @@ use std::path::PathBuf;
 use anstream::println;
 use anyhow::{bail, Result};
 use pretty_assertions::StrComparison;
-use schemars::schema_for;
+use schemars::{schema_for, JsonSchema};
+use serde::Deserialize;
 
 use uv_workspace::Options;
 
 use crate::ROOT_DIR;
 
+#[derive(Deserialize, JsonSchema)]
+#[serde(deny_unknown_fields)]
+#[allow(dead_code)]
+// The name is used in the schema
```

Comment:
What does this comment mean? I think in general we should work on including fewer of these comment fragments in the PR. Comments need to stand alone and carry meaning well into the future.